opba-protocols/xs2a-protocol/src/main/java/de/adorsys/opba/protocol/xs2a/entrypoint/ais/Xs2aListTransactionsEntrypoint.java

```diff
@@ -40,13 +40,17 @@
     private final ExtendWithServiceContext extender;
 
     @Override
-    public CompletableFuture<Result<TransactionListBody>> execute(ServiceContext<ListTransactionsRequest> serviceContext) {
+    public CompletableFuture<Result<TransactionsResponseBody>> execute(ServiceContext<ListTransactionsRequest> serviceContext) {
+        TransactionListXs2aContext context = mapper.map(serviceContext.getRequest());
```

Comment:
These 3 lines are not needed, they are replaced by `prepareContext(serviceContext)`

Reply:
missed it during rebase